<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperatorConsoleDialog</class>
 <widget class="QDialog" name="OperatorConsoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imatest IT Operator Console</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="imageColumn">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QWidget" name="imageDisplayWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="image">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>561</width>
           <height>411</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="qsoHoritzontalLayout">
          <item>
           <layout class="QVBoxLayout" name="qsoLeftVerticalLayout">
            <item>
             <widget class="QStackedWidget" name="qsoUL">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="cursor">
               <cursorShape>BlankCursor</cursorShape>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_3">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="ulpassed">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>O</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_4">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="ulfailed">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QStackedWidget" name="qsoLL">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_5">
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="llPassed">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>O</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_6">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="llFailed">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QStackedWidget" name="qsoCenter">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page_1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="centerPassed">
                <property name="font">
                 <font>
                  <pointsize>48</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 255, 0);</string>
                </property>
                <property name="text">
                 <string>O</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="centerFailed">
                <property name="font">
                 <font>
                  <pointsize>48</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 0, 0);</string>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="qsoRightVerticalLayout">
            <item>
             <widget class="QStackedWidget" name="qsoUR">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="passed_2">
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="labelPassed_3">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>O</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="failed_2">
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="labelFailed_3">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QStackedWidget" name="qsoLR">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_7">
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="lrPassed">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(0, 255, 0);</string>
                  </property>
                  <property name="text">
                   <string>O</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_8">
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="llFailed_2">
                  <property name="font">
                   <font>
                    <pointsize>48</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: rgb(255, 0, 0);</string>
                  </property>
                  <property name="text">
                   <string>X</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="labelFps">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>480</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>0.00 fps</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <zorder>image</zorder>
        <zorder>layoutWidget</zorder>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="moveableWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="logo">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Data/imatest_logo.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="font">
         <font>
          <pointsize>24</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="reasonText"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="summaryText"/>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="logText"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radioBlemish">
            <property name="text">
             <string>Blemish</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioSfr">
            <property name="text">
             <string>SFRplus</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioCharts">
            <property name="text">
             <string>Arbitrary Charts</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="buttonGridLayout">
          <item row="1" column="1">
           <widget class="QPushButton" name="btnStop">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="btnJSON">
            <property name="text">
             <string>Show JSON</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btnPassFail">
            <property name="text">
             <string>Set Pass/Fail</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="btnStart">
            <property name="text">
             <string>Start</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnQuit">
            <property name="text">
             <string>Quit</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnSetup">
            <property name="text">
             <string>Setup</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>moveableWidget</zorder>
  <zorder>verticalWidget</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
